File: app/src/main/java/com/sharpdroid/registroelettronico/Views/SubjectDetails/WeeklyScheduleView.java
package com.sharpdroid.registroelettronico.Views.SubjectDetails;

import android.content.Context;
import android.support.v7.widget.CardView;
import android.util.AttributeSet;

public class WeeklyScheduleView extends CardView {
    Context mContext;

    public WeeklyScheduleView(Context context) {
        super(context);
        this.mContext = context;
    }

    public WeeklyScheduleView(Context context, AttributeSet attrs) {
        super(context, attrs);
        this.mContext = context;
    }

    public WeeklyScheduleView(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
        this.mContext = context;
    }
}

File: app/src/main/java/com/sharpdroid/registroelettronico/views/subjectDetails/WeeklyScheduleView.kt
package com.sharpdroid.registroelettronico.views.subjectDetails

import android.content.Context
import android.support.v7.widget.CardView
import android.util.AttributeSet

class WeeklyScheduleView : CardView {
    internal var mContext: Context

    constructor(context: Context) : super(context) {
        this.mContext = context
    }

    constructor(context: Context, attrs: AttributeSet) : super(context, attrs) {
        this.mContext = context
    }

    constructor(context: Context, attrs: AttributeSet, defStyleAttr: Int) : super(context, attrs, defStyleAttr) {
        this.mContext = context
    }
}
